app/lib/clients/create_tweet.rb

```diff
@@ -49,21 +59,17 @@ def text
   end
 
   def status
-    if @tweet.attrs[:retweeted]
+    if !@tweet.attrs[:retweeted_status].nil?
```

Comment:
`@tweet.attrs[:retweeted_status]` or `@tweet.attrs.has_key(:retweeted_status]`

This and attributes would be a great place for some test coverage.